counterpartylib/test/fixtures/vectors.py

```diff
@@ -1239,6 +1285,10 @@
             'out': ('mn6q3dS2EnDUx3bmyWc6D4szJNVGtaR7zc',
                     [('mtQheFaSfWELRB2MyMBaiWjdDm6ux9Ezns', None)],
                     bytes.fromhex('0000000001530821671b10650000000005f5e100'))
+        }, {
+            'comment': 'send to a REQUIRE_MEMO address',
+            'in': (ADDR[0], ADDR[6], 'XCP', DP['quantity']),
+            'error': (exceptions.ComposeError, '[\'destination requires memo\']')
```

Comment:
I don't see any tests that cover the `enhanced_send.parse` and `send1.parse` methods.  Did I miss them?  If not, please add tests here as this is sensitive code for maintaining consensus.

We should cover both successful and failed sends to an address with MEMO_REQUIRED set.

Reply:
Gotcha... will add the coverage for parse methods.... didn't catch that as i'm kinda new to this kind of procedure heh